addons/project/static/src/components/subtask_kanban_list/subtask_kanban_list.js

```diff
@@ -12,6 +12,17 @@ import { standardWidgetProps } from "@web/views/widgets/standard_widget_props";
 export class SubtaskKanbanList extends Component {
     setup() {
         this.actionService = useService("action");
+        this.orm = useService("orm");
+        this.subTaskIds = [];
```

Comment:
is this necessary? :thinking:

Reply:
Or is it not? That is the question... :monocle_face:

Reply:
the variable or the initialization ? 
cc @pkyriakou